src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
-body {
+@import "./utils/variables";
+@import "./utils/mixins";
+
+.page {
   margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 726px;
```

Comment:
Hey, great job! In my opinion everithing is fine. Almost:))
It's just my opinion, but making exact width isn't the best idea. What if somebody decide to change day cells size?